Gemfile.lock

```diff
@@ -16,3 +16,6 @@ DEPENDENCIES
   attr_comparable!
   minitest
   rake (>= 0.9)
+
+BUNDLED WITH
+   1.12.5
```

Comment:
I can pull it out manually, but it seems to have inserted automatically. I guess if it's the only change to the file then adding doesn't have much value, but it's just going to be added back in the future